File: SooYeon/240101_week/Programmers_147355_크기가 작은 부분 문자열.js
function solution(t, p) {
    var answer = 0;
    let arr = [];
    
    for(let i = 0; i <= t.length - p.length; i++) {
        let number = '';
        
        for(let j = i; j < p.length + i; j++) {
            number += t[j]
        }
        if(Number(number) <= Number(p)) { 
            arr.push(number);
        }
    }
    
    answer = arr.length
    
    return answer;
}

File: SooBin/240101_week/programmers_147355_크기가 작은 부분문자열.cpp
#include <string>
#include <vector>

using namespace std;

int solution(string t, string p) {
    int answer = 0;
    for(int i = 0; i < t.size() - p.size() + 1; i++){
        string temp = "";
        for(int j = 0; j < p.size(); j++){
            temp += t[i+j];
        }
        if(temp  <= p) answer++;
        temp = "";
    }
    return answer;
}